pyproject.toml

```diff
@@ -126,5 +126,6 @@ messages_control.disable = [
   "line-too-long",
   "missing-module-docstring",
   "wrong-import-position",
+  "abstract-class-instantiated"
```

Comment:
ooh where was this coming from?

Reply:
oddly it was using the `pd.ExcelWriter` which I couldn't be bothered to then deal with 😆

Reply:
weird since you've used it here exactly as suggested in the [pandas docs](https://pandas.pydata.org/docs/reference/api/pandas.ExcelWriter.html) - fine to add this in then!

Reply:
Yeah right, my guess is that there's some magic under the hood which creates your writer with the specific engine that you're asking for.